test/geometry/test_dsnt.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+import kornia as kornia
+from test.common import dtype, device
+
+import torch
+from torch.testing import assert_allclose
+
+
+class TestRenderGaussian2d:
+    @pytest.fixture
+    def gaussian(self, device, dtype):
+        return torch.tensor([
+            [0.002969, 0.013306, 0.021938, 0.013306, 0.002969],
+            [0.013306, 0.059634, 0.098320, 0.059634, 0.013306],
+            [0.021938, 0.098320, 0.162103, 0.098320, 0.021938],
+            [0.013306, 0.059634, 0.098320, 0.059634, 0.013306],
+            [0.002969, 0.013306, 0.021938, 0.013306, 0.002969],
+        ], dtype=dtype, device=device)
+
+    def test_pixel_coordinates(self, gaussian, device, dtype):
+        mean = torch.tensor([2.0, 2.0], dtype=dtype, device=device)
+        std = torch.tensor([1.0, 1.0], dtype=dtype, device=device)
+        actual = kornia.geometry.dsnt.render_gaussian_2d(mean, std, (5, 5), False)
+        assert_allclose(actual, gaussian, rtol=0, atol=1e-4)
+
+    def test_normalized_coordinates(self, gaussian, device, dtype):
+        mean = torch.tensor([0.0, 0.0], dtype=dtype, device=device)
+        std = torch.tensor([0.25, 0.25], dtype=dtype, device=device)
+        actual = kornia.geometry.dsnt.render_gaussian_2d(mean, std, (5, 5), True)
+        assert_allclose(actual, gaussian, rtol=0, atol=1e-4)
+
+    def test_jit_trace(self, device, dtype):
+        def op(mean, std):
+            return kornia.geometry.dsnt.render_gaussian_2d(mean, std, (5, 5), True)
+        mean = torch.tensor([0.0, 0.0], dtype=dtype, device=device)
+        std = torch.tensor([0.25, 0.25], dtype=dtype, device=device)
+        args = (mean, std)
+        op_jit = torch.jit.trace(op, args)
+        assert_allclose(op(*args), op_jit(*args), rtol=0, atol=1e-5)
```

Comment:
add test for  jit script too

Reply:
I couldn't get jit.script to work with `render_gaussian_2d` due to the tuple and bool arguments.

Reply:
I've recently seen that torchvision  started using torch.jit.annotations.
Maybe this could be related to the issue: https://github.com/pytorch/vision/blob/master/torchvision/models/detection/rpn.py#L12

Reply:
Oops, my mistake, the issue wasn't actually the argument types, it was something else. I think I can figure it out now though.